docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`-` Convex staking and Curve gauge balances will no longer need a manual balances referesh to be detected.
+
```

Comment:
luki in his PR added already one. Maybe coordinate with him to avoid double adding it